src/modules/Hosts/HostsUILib/HostsMainPage.xaml

```diff
@@ -511,14 +511,15 @@
                                 Grid.Column="4"
                                 Width="40"
                                 MinWidth="0"
-                                HorizontalAlignment="Right"
+                                HorizontalAlignment="Center"
                                 GotFocus="Entries_GotFocus"
                                 IsOn="{x:Bind Active, Mode=TwoWay}"
                                 OffContent=""
                                 OnContent="" />
                             <Button
                                 x:Uid="DeleteEntryBtn"
                                 Grid.Column="5"
+                                Width="48"
```

Comment:
Why is this needed? This make delete button wider than the buttons on top.
![image](https://github.com/microsoft/PowerToys/assets/25966642/b11d5f1d-74ec-4ce0-baff-9bc8f2653323)
![image](https://github.com/microsoft/PowerToys/assets/25966642/5031e2e5-2096-45e0-941c-737ca4cff623)

Reply:
Need to check. Fid this to fix the vertical alignment problem. But this was before I found and removed the empty column on the right side.

Reply:
Removed